addons/l10n_be_pos_restaurant/data/template/account.tax-be.csv

```diff
@@ -0,0 +1,5 @@
+"id","sequence","description","invoice_label","name","amount","amount_type","type_tax_use","tax_group_id","tax_scope","active","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/tag_ids","repartition_line_ids/account_id","repartition_line_ids/factor_percent","name@fr","name@nl"
+"tax_alcohol_luxury","10","21% VAT (Alcohol, luxury)","21%","21% Alcohol / luxury","21.0","percent","sale","tax_group_tva_21","","","base","invoice","+03","","","21%","21%"
+"","","","","","","","","","","","tax","invoice","+54","a451","","",""
+"","","","","","","","","","","","base","refund","+49","","","",""
+"","","","","","","","","","","","tax","refund","+64","a451","","",""
```

Comment:
We also create a "take away" fiscal position that reduces 21% and 12% into 6% taxes.
We create this new tax because we don't want this reduction to be applied on alcohol products (whenever it's a takeaway or not).